activerecord/lib/arel/visitors/mysql.rb

```diff
@@ -13,9 +13,10 @@ def visit_Arel_Nodes_UnqualifiedColumn(o, collector)
           visit o.expr, collector
         end
 
-        ###
         # :'(
-        # https://dev.mysql.com/doc/refman/8.0/en/select.html#id3482214
+        # To retrieve all rows from a certain offset up to the end of the result set,
+        # you can use some large number for the second parameter
+        # https://dev.mysql.com/doc/refman/en/select.html
```

Comment:
In this case, the id in the comment was not working anymore. I opted to copy the relevant manual paragraph in the comment, because there was nothing close that I could target to point to the right section.